src/qz/printer/status/CupsUtils.java

```diff
@@ -0,0 +1,223 @@
+package qz.printer.status;
+
+import com.sun.jna.Pointer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import sun.net.util.URLUtil;
```

Comment:
`sun.net.util.URLUtil` is deprecated.  Is it needed or is this leftover from something that was done previously?

Reply:
I will look into alternatives.